types/index.d.ts

```diff
@@ -436,7 +439,12 @@ export type Admin = {
     groupId: string
     topic: string
     resolveOffsets?: boolean
-  }): Promise<Array<SeekEntry & { metadata: string | null }>>
+  }): Promise<FetchOffsetPartitions>
```

Comment:
This isn't correct, is it? This indicates that the returned promise resolves to something like:

```js
{
  partitions: [{ partition: '1', offset: '123', metadata: null }]
}
```

But unless I'm mistaken, it actually resolves to:

```js
[{ partition: '1', offset: '123', metadata: 'foo' }]
```

I guess this type should be unchanged from before:

```ts
Array<SeekEntry & { metadata: string | null }>
```

Btw, prefer to use `PartitionOffset` over `SeekEntry`. They are aliases of each other, so functionally there's no difference, but eventually `SeekEntry` will be deprecated in favor of `PartitionOffset`, so it's good if we don't spread it further.

Reply:
Right! @Nevon my mistake. Fixed that.